diff --git a/application/src/main/java/run/halo/app/core/counter/CounterServiceImpl.java b/application/src/main/java/run/halo/app/core/counter/CounterServiceImpl.java
index cea0ea7c2..875a20b90 100644
--- a/application/src/main/java/run/halo/app/core/counter/CounterServiceImpl.java
+++ b/application/src/main/java/run/halo/app/core/counter/CounterServiceImpl.java
@@ -27,7 +27,7 @@ public class CounterServiceImpl implements CounterService {
     }
 
     @Override
-    public Mono<Counter> getByName(String counterName) {
+    public synchronized Mono<Counter> getByName(String counterName) {
         return client.fetch(Counter.class, counterName);
     }
